package.json

```diff
@@ -53,14 +53,14 @@
     "grunt-contrib-copy": "^0.7.0",
     "grunt-contrib-jshint": "^0.11.2",
     "grunt-contrib-uglify": "^0.7.0",
-    "grunt-jsdoc": "^0.5.8",
+    "grunt-jsdoc": "^2.0.0",
```

Comment:
@adon-at-work will you please take a look? the issues involved here are security related with the package dependencies